Source/WebCore/css/MediaList.idl

```diff
@@ -31,9 +33,8 @@
 ] interface MediaList {
     stringifier attribute [LegacyNullToEmptyString] DOMString mediaText;
     readonly attribute unsigned long length;
-
     getter DOMString? item(unsigned long index);
-
+    // FIXME: Rename 'oldMedium' and 'newMedium' as 'medium'.
```

Comment:
You could rename this directly here too.

Reply:
Done in `[b8e6b2a](https://github.com/WebKit/WebKit/commit/b8e6b2a0614bd424624ac1c83dd089d159bb38a5)`.